food-truck-frontend/src/pages/truck/edit/[truckId].tsx

```diff
@@ -78,6 +78,7 @@ class EditTruck extends Component<TruckProps, TruckComponentState> {
   }
 
   onSubmit = (formData: any, response: AxiosResponse) => {
+    api.post(`/reindex`).catch();
```

Comment:
Why don't we have the backend service call the reindexing method, instead of having the frontend need to know about indexing?

Also, having an empty catch will swallow any exceptions that get thrown--usually not the best idea.